### ISSUE TITLE
Provide an API to get the bounding box of a cube

### ISSUE THREAD

**Issue by Cadair** [sunpy/ndcube#200]
It would be useful to be able to get some representation of the extent of the cube along various axes (especially spatial) to over plot on other images. i.e. plotting the field of view of IRIS on an AIA image.

**Comment by DanRyanIrish** [comment]
How is this done in other classes, like ```Map```?

**Comment by Cadair** [comment]
I don't think it really is. For almost all solar data, you can represent a spatial bounding box with a top left and bottom right coordinate.

**Comment by DanRyanIrish** [comment]
So how would like work?  Output a new axes object representing the FOV?

**Comment by Cadair** [comment]
yeah some object to represent the box, not really sure on the details.
